File: examples/automations/mailboxes/update.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.production")

reclient = Retab()

automation = reclient.processors.automations.links.update(
    link_id="lnk_Xf15nXFpo7mwfGT1aSYo4",
    name="Link Automation Updated",
)

print(automation.model_dump_json(indent=2))

File: examples/typescript/automations/mailboxes/update.ts
import { config } from 'dotenv';
import { Retab } from '@retab/node';



async function main() {
  config({ path: '../../../.env.production' });
  
  const reclient = new Retab();
  
  const automation = await reclient.processors.automations.mailboxes.update({
    mailbox_id: "mb_FRf6FX5fYkenZ_JJlL5GD",
    name: "Mailbox Automation Updated",
  });
  
  console.log(JSON.stringify(automation, null, 2));
  
}

main().catch(console.error);